src/views/document/RouteView.vue

```diff
@@ -198,13 +204,17 @@ export default {
 
     showMissingHistoryBanner() {
       const doc = this.document;
+      if (this.version || ['empty', 'draft'].includes(doc.quality)) {
+        // hide notice if we are consulting an old version, or if low quality doc banner is already shown
```

Comment:
I don't like the fact that the logic to know whether or not a banner for low quality should be displayed is duplicated here, but didn't find any obvious way before going back to my real work...

Reply:
Why not moving `lowQuality` property from LowDocumentQualityBanner component to here ? 

Line 78 would be : 

```
<low-document-quality-banner v-if="showLowQualityBanner" :document="document" />
```

and line 207 would be : 

```
// do not show the two banners
if(this.showLowQualityBanner) { return false; } 
```

Side note, `showLowQualityBanner` should be false for versioned view.

Reply:
yeah, thought about that too, but I was reluctant because I assumed the responsibility should be that of the component itself. And we would still duplicate the logic for route and article views.

but maybe it's best nonetheless, and we can assume the decision may be that of the view (in which case we could add the warning to other documents with different criteria?)

Reply:
> Side note, showLowQualityBanner should be false for versioned view.

This is the case already, isn't it?

Reply:
>  I assumed the responsibility should be that of the component itself.

Vaste question :) Usually, when it comes to be entirely displayed, I see these situations : 

1. the decision's logic can change => it's up to the parent to decide
2. the decision's logic is always the same, and is very complex => set it in the child
3. the decision's logic is simple => Do whatever you want :) 
4. you need the decision's result in the parent =>  do not duplicate, set it in the parent.
5.  you need the decision's result in the parent, and it's complex => mixin


Other solution, (but I do not like it, because it causes complex behaviors) : `if (this.$refs.lowQualityBanner.visible)`

> This is the case already, isn't it?

Nope, or did I missed somethin' ? 

```
    lowQuality() {
      return ['empty', 'draft'].includes(this.document.quality);
    },
```

Reply:
> Nope, or did I missed somethin' ?
Yes, there is a test on `<low-quality-banner>`tag to check if `version` is truthy. But maybe it should move with the whole logic. I'll fix this according to your comments thanks.

What about i18n?

Reply:
> What about i18n?

We hit the limit of our system. 

You assume the in all our langs, "Feel free to" will be before "edit". I bet one :beer: it's not true. 

A hack would be to make the entire sentence a link:

```
    <edit-link show-always :document="document" :lang="lang">
      <span v-translate>Feel free to edit</span>
    </edit-link>
```

(I removed the `translate-context`, IMO it's useless here)